packages/design-system/src/components/ChoiceList/Choice.tsx

```diff
@@ -112,13 +116,16 @@ export const Choice = (props: ChoiceProps) => {
 
   let errorId;
   let errorElement;
-  if (props.errorMessage) {
+  if (!props._choiceChild) {
     errorId = props.errorId ?? `${props.id}__error`;
```

Comment:
I think this assignment statement being in a different kind of condition now might be one clue about what's happening